app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.ts

```diff
@@ -249,11 +249,28 @@ export class AwsLoadBalancerTransformer {
             action.targetGroupName = action.targetGroupName.replace(`${applicationName}-`, '');
           });
 
+          // Remove the default rule because it already exists in defaultActions
+          listener.rules = (listener.rules || []).filter((l) => !l.default);
+          listener.rules.forEach((rule) => {
+            (rule.actions || []).forEach((action) => {
+              action.targetGroupName = action.targetGroupName.replace(`${applicationName}-`, '');
+            });
+            rule.conditions = rule.conditions || [];
+          });
+
+          // Sort listener.rules by priority.
+          listener.rules.sort((a, b) => {
+            if (a.priority < b.priority) { return -1; }
+            if (a.priority > b.priority) { return 1; }
+            return 0;
```

Comment:
Can you just `return a.priority - b.priority`? or is this taking into account `'default'`? If it's the latter, might be worth a comment explaining the expected sort order (it's not clear to me what will happen if priority is `'default'`)